src/client/NuGet.CommandLine.StagingExtensions/StageManagementResource.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Client.Staging;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.v3;
+
+namespace NuGet.CommandLine.StagingExtensions
+{
+    public class StageManagementResource : Protocol.Core.Types.INuGetResource

```

Comment:
done

---
In reply to: [66826446](https://github.com/NuGet/NuGet.Staging/pull/62#discussion_r66826446) [](ancestors = 66826446)